### PR TITLE
Add Elemental Sforzo placeholder, to make the ability usable.

### LINKED ISSUE #429: Adding Elemental Sforzo to ability list
**_I have:_**

- [x] searched existing issues (http://github.com/project-topaz/topaz/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Elemental Sforzo is already coded in the status effect list and is available when you use godmode, but the ability itself is not coded as an ability for runefencer to use natively. 

-----------------------------------
-- Ability: Elemental Sforzo
-- Grants immunity to all magical attacks.
-- Obtained: Runefencer Level 1
-- Recast Time: 1:00:00
-- Duration: 0:00:30
-----------------------------------
require("scripts/globals/settings")
require("scripts/globals/status")
-----------------------------------

function onAbilityCheck(player,target,ability)
    return 0,0
end

function onUseAbility(player,target,ability)
    player:addStatusEffect(tpz.effect.ELEMENTAL_SFORZO,1,0,30)
end

### PR DESCRIPTION
**Pull request by mrhappyasthma**
A few things are still needed:

1. Add enmity when cast.
2. Update the status_effect to also protect against status effect
spells that cause debuffs.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by TeoTwawki** [comment]
>spell castable

*ability usable

:)

File: scripts/globals/abilities/elemental_sforzo.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onAbilityCheck(player,target,ability)
+  return 0,0
+end
+
+function onUseAbility(player,target,ability)
+  -- TODO: Add enmity when cast.
```
Comment (TeoTwawki) [inline scripts/globals/abilities/elemental_sforzo.lua]: there are fields in the db table for the CE and VE, if the amounts are known

Comment (mrhappyasthma) [reply scripts/globals/abilities/elemental_sforzo.lua]: Could you clarify what CE and VE stand for?

Comment (TeoTwawki) [reply scripts/globals/abilities/elemental_sforzo.lua]: Cumulative Enmity and Volatile Enmity. One of them decays over time, the other does not.

https://www.bg-wiki.com/index.php?curid=11855#Cumulative_Enmity_Loss

https://github.com/project-topaz/topaz/blob/release/sql/abilities.sql#L28

Comment (mrhappyasthma) [reply scripts/globals/abilities/elemental_sforzo.lua]: Looks like those values aren't populated: https://github.com/project-topaz/topaz/blob/release/sql/abilities.sql#L293

It's listed as `1800` CE and `7200` VE here: 
https://www.bg-wiki.com/bg/Elemental_Sforzo

Comment (TeoTwawki) [reply scripts/globals/abilities/elemental_sforzo.lua]: That's why I was telling you where to populate them.

**Review by ibm2431 — CHANGES_REQUESTED** [review]
Thanks, mrhappyasthma!

Could you add the CE and VE to the abilities.sql?

Indenting should be 4 spaces; the indents here are two~

**Comment by mrhappyasthma** [comment]
Whoops. Force of habit. Will update the spacing.

Is it sufficient to just add the CE/VE to the sql file? Or do I need to apply those enmity values somewhere in code?

It seems like they will get picked up in the core and added to the `spell`. But I couldn't seem to find where it's applied to the player (if that's done automatically).

Any pointers would be helpful :) thanks!

**Comment by TeoTwawki** [comment]
> Is it sufficient to just add the CE/VE to the sql file? Or do I need to apply those enmity values somewhere in code?

Yep just plug into the table, core does the rest on [ability ](https://github.com/project-topaz/topaz/blob/8985d2af68876367592222836db1f8e5b15c8175/src/map/ai/states/ability_state.cpp#L76)use.

File: scripts/globals/abilities/elemental_sforzo.lua (hunk 1 above)
Comment (mrhappyasthma) [reply scripts/globals/abilities/elemental_sforzo.lua]: Done. Please take another look.

**Comment by mrhappyasthma** [comment]
Updated indentation and added the CE/VE values.

Thanks for the code pointers.

**Review by ibm2431 — APPROVED** [review]
Thanks, mrhappyasthma!

**Pull request merged**